src/pages/hcp/api-docs/vault-secrets/[[...page]].tsx

```diff
@@ -123,7 +123,7 @@ export const getStaticProps: GetStaticProps<
 	 * 🚩 If the flag is enabled, use the revised template
 	 */
 	if (USE_REVISED_TEMPLATE) {
-		return await getOpenApiDocsStaticProps({ params })
+		return await getOpenApiDocsStaticProps(PRODUCT_SLUG, { params })
```

Comment:
We pass `PRODUCT_SLUG` here since we need to generate product-specific mobile menu panes to render in the view.